From c0ce702a5dc704287cdcf7f87f6cb10c3ad3da63 Mon Sep 17 00:00:00 2001
From: Sergio Pena <sergio.pena@cloudera.com>
Date: Thu, 10 Dec 2015 13:21:37 -0600
Subject: [PATCH 0431/1164] CLOUDERA-BUILD: CDH-35381: Hive pre-commit test is
 not using -Dmaven.repo.local when running .q
 tests causing different output results

Change-Id: I3b154901865a3345178db8b799109f02f88f2281
---
 cloudera/pre_commit_hook.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cloudera/pre_commit_hook.sh b/cloudera/pre_commit_hook.sh
index 274635d..e43438d 100755
--- a/cloudera/pre_commit_hook.sh
+++ b/cloudera/pre_commit_hook.sh
@@ -76,6 +76,6 @@ if [ -n "$tests_modified" ]; then
     driver=`echo $t | cut -d: -f1`
     files=`echo $t | cut -d: -f2`
 
-    mvn test -Phadoop-2 -Dtest=$driver -Dqfile=$files
+    mvn test -Phadoop-2 -Dmaven.repo.local="$MVN_REPO_LOCAL" -Dtest=$driver -Dqfile=$files
   done
 fi
-- 
1.7.9.5

